modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/ClassicEditor.js

```diff
@@ -66,7 +66,7 @@ const ClassicEditor = React.forwardRef(
 		}, [contents]);
 
 		const onChangeCallback = () => {
-			if (!onChangeMethodName && !onChange) {
+			if (!onChangeMethodName && !onChange || !editorRef.current) {
```

Comment:
> Passing a stable reference or checking against null before assigning the ref may do the trick

I'm just checking against null, it can be okay on our side? Considering the first point(passing a stable reference) need to be done by the product team that is maintaining the place where the ClassicEditor is being used?